PCAxis.Web.Controls/VariableSelector/VariableSelectorValueSelectCodebehind.vb

```diff
@@ -406,9 +407,9 @@ Public Class VariableSelectorValueSelectCodebehind
 
         'Match grouping selected in the interface. Use of backbutton may have caused missmatches
         'between model on server and sclections sent by the interface.
-        If GroupingDropDown.SelectedValue.Length > 0 Then
-            Marker.SelectedGrouping = GroupingDropDown.SelectedValue
-        End If
+        'If GroupingDropDown.SelectedValue.Length > 0 Then
```

Comment:
Remove the code instead of comment it out.

Reply:
Comments removed